build.gradle

```diff
@@ -1,29 +1,26 @@
 plugins {
-    id "java-library"
-    id "scala"
-    id "maven-publish"
-    id 'fabric-loom' version '0.10.30'
-    id "com.matthewprenger.cursegradle" version "1.4.0"
+	id 'java-library'
+	id 'scala'
+	id 'maven-publish'
+	id 'fabric-loom' version '1.5-SNAPSHOT'
 }
 
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
-
 archivesBaseName = project.archives_base_name
 group = project.maven_group
 
-version = mod_version
-def ENV = System.getenv()
-version = version + "+scala." + project.scala_version + (ENV.GITHUB_ACTIONS ? "" : ".local")
+version = mod_version + "+scala." + project.scala_version + (System.getenv('GITHUB_ACTIONS') ? "" : ".local")
 
 dependencies {
-    minecraft "com.mojang:minecraft:${project.minecraft_version}"
-    mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+	minecraft "com.mojang:minecraft:${project.minecraft_version}"
+	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
-	include(api("org.scala-lang:scala3-library_3:${project.scala_version}"))
-	// Depended on by scala3
-	include(api("org.scala-lang:scala-library:2.13.6"))
+	// Scala Dependencies
+	implementation "org.scala-lang:scala3-library_3:${project.scala_version}"
+	implementation "org.scala-lang:scala-reflect:${project.scala_reflect_version}"
+	implementation "org.scala-lang.modules:scala-xml_3:${project.scala_xml_version}"// might be unnecessary but wtv
```

Comment:
Is this needed or not? I dont see why it would be, why also add `scala-reflect`?

Reply:
adding scala reflect can add more complex coding practice when using scala. and the xml idk why i added that i think it was just late at night so i wasnt thinking.